resources/grafana/dashboards/machines.json

```diff
@@ -941,8 +941,8 @@
             "measurement": "total-anomaly",
             "orderByTime": "ASC",
             "policy": "default",
-            "query": "SELECT mean(/^$Sensor$/) as \"field\" FROM \"original-input\" WHERE (\"machine\" =~ /^$Machine$/) AND $timeFilter GROUP BY time($__interval) fill(null)",
-            "rawQuery": false,
+            "query": "SELECT mean(*) FROM \"total-anomaly-scaled\" WHERE (\"machine\" =~ /^$Machine$/) AND $timeFilter GROUP BY time($__interval) fill(null)",
```

Comment:
IDK if this is intended. Talk to @flikka , he is an expert on grafana-influx-query-shitups

Reply:
That's probably on me, as I merely _looked_ at a query in influx and broke this. :laughing: